Adding conditional variables and renotify option (#6711)
DataDog/documentation: content/en/synthetics/api_tests/_index.md

A notification is sent according to the set of alerting conditions. To configure notifications:

1. Select users and/or [services][4] to send the notifications to. Note that you can use [`@-notification`][5] in the **message** field, similarly to monitors.

> 2. Enter a **message** for the API test. This field allows standard [Markdown formatting][6] and supports the below [conditional variables][7]:

Reviewer: ```suggestion
2. Enter a **message** for the API test. This field allows standard [Markdown formatting][6] and supports the following [conditional variables][7]:
```

    | Conditional Variable       | Description                                                         |
    |----------------------------|---------------------------------------------------------------------|
    | `{{#is_alert}}`            | Show when monitor alerts                                            |
    | `{{^is_alert}}`            | Show unless monitor alerts                                          |
    | `{{#is_recovery}}`         | Show when monitor recovers from either ALERT   |
    | `{{^is_recovery}}`         | Show unless monitor recovers from either ALERT |

    Notification messages include the **message** defined in this section and information about which assertion failed and why.

> 3. Specify a renotification frequency. If you don't want to be renotified in case your test keeps failing, leave the option to `Never renotify if the monitor has not been resolved`.

Reviewer: ```suggestion
3. Specify a renotification frequency. To prevent renotification for constantly failing tests, leave the option as `Never renotify if the monitor has not been resolved`.
```

Author: "Constantly failing tests" sounds like tests that are set to fail all the time to me. What about just:
```suggestion
3. Specify a renotification frequency. To prevent renotification for failing tests, leave the option as `Never renotify if the monitor has not been resolved`.
```

Reviewer: sounds good

> 4. Click **Save** to save your API test.

Reviewer: ```suggestion
4. Click **Save**.
```
